pkg/controller/networkaddonsconfig/networkaddonsconfig_controller.go

```diff
@@ -710,3 +719,30 @@ func isOpenshiftSingleReplica(c k8sclient.Client) (bool, error) {
 
 	return infraConfig.Status.InfrastructureTopology == osconfv1.SingleReplicaTopologyMode, nil
 }
+
+func isPodSecurityEnabled(c k8sclient.Client) (bool, error) {
+	apiPod, err := apiServerPod(c)
+	if err != nil {
+		return false, err
+	}
+
+	for _, cmd := range apiPod.Spec.Containers[0].Command {
+		if strings.Contains(cmd, "PodSecurity=true") {
```

Comment:
When this is no longer an alpha feature, could it happen that this would be enabled by default without the need of the explicit featuregate?
I'm asking because in that case, we may miss detecting `PodSecurity` to be enabled

Reply:
It could happen that it will be enabled by default. The thing is by now CNAO can be applied on k8s versions lower than 1.22, and the API might not exist. I could not think of a better way to discover this.
Do you have an idea?

Perhaps when it is not alpha anymore it will appear in the `- --enable-admission-plugins` section but this is just speculation